lectures/matplotlib.md

```diff
@@ -352,7 +352,55 @@ draw_graphs(style='dark_background')
 
 You can use the function to experiment with other styles in the list.
 
-If you are interested, you can even create [your own style sheets](https://matplotlib.org/stable/tutorials/introductory/customizing.html#defining-your-own-style).
+If you are interested, you can even create your own style sheets.
+
+Paraeters for your style sheets are stored in a dictionary-like variable `plt.rcParams`
+
+```{code-cell} python3
+---
+tags: [hide-output]
+---
+ 
+print(plt.rcParams.keys())
+
+```
+
+There are many parameters you could set for your style sheets.
+
+Set parameters for your style sheet by: 
+
+1. creating your own [`matplotlibrc` file](https://matplotlib.org/stable/tutorials/introductory/customizing.html#defining-your-own-style), or
+2. updating values stored in the dictionary-like variable `plt.rcParams`
+
+Let's change the style of our overlaid density lines
+
+```{code-cell} python3
+from cycler import cycler
+
+# set to the default style sheet
+plt.style.use('default')
+
+# update linewidth
+plt.rcParams['lines.linewidth'] = 2
+# add horizontal grid lines
+plt.rcParams['axes.grid'] = True 
+plt.rcParams['axes.grid.axis'] = 'y'
+# update colors for density lines
+plt.rcParams['axes.prop_cycle'] = cycler('color', ['dimgray', 'slategrey', 'darkgray'])
+
```

Comment:
It might be worth splitting this into two code cells and add a sentence in between that let's people know these settings are `global` in nature and can be applied to any plot generated after the `rcParams` has been set? What do you think @HumphreyYang

Reply:
Thanks @mmcky, I will add a sentence to point out this important fact.